src/routes/Project/components/Table.js

```diff
@@ -149,14 +155,16 @@ export default class Table extends Component {
     const tableDiv = document.getElementById('table-component');
 
     let oldClip = this.props.tracks[oldTrackNum].clips[clipId];
-    let newClip = { ...oldClip };
+    let newClip = _.cloneDeep(oldClip);
```

Comment:
I suppose its fine for clip since its such a shallow object already, would it even make a difference since the object actually has no depth that I can think of? be careful with deep cloning avoid at all costs, its pretty intensive operation.